presto-main/src/main/java/com/facebook/presto/Session.java

```diff
@@ -41,6 +41,7 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.TimeZone;
+import java.util.stream.Collectors;
```

Comment:
`Application of defaults during session creation conflicts with resource
group selection and query parsing. Separating out the application of
default session properties allows us to refactor query creation in
SqlQueryManager.`